src/test/todoObjectClass.test.js

```diff
@@ -24,24 +25,59 @@ describe('add and remove tasks', () => {
   });
 
   test('add two task', () => {
-    document.body.innerHTML = `
-            <label for="add" class="add"></label>
-            <input 
-                name="add"
-                id="add"
-                placeholder="Add to your list.."
-            />
-            <ul class="listContainer"></ul>`;
-
-    newItem.addNewToDo(mainArr, 'Go shopping');
+    newItem.addNewToDo(mainArr, 'Swimming lesson');
     newItem.addNewToDo(mainArr, 'Practice Unit Testing');
     const listContainer = document.querySelectorAll('.listContainer li');
-    expect(listContainer).toHaveLength(2);
+    expect(listContainer).toHaveLength(3);
   });
 
   test('Remove one task', () => {
     newItem.removeFromToDo(1);
     const listContainer = document.querySelectorAll('.listContainer li');
     expect(listContainer).toHaveLength(2);
   });
+});
+
+describe('Edit task', () => {
+  test('edit task', () => {
+    // localStorage.clear();
+    newItem.addNewToDo(mainArr, 'Go Mall');
+    newItem.editItem(3, 'Go Market');
+    const MyToDo = new MyToDoList();
+    const val = MyToDo.tasks[2].description;
+    expect(val).toBe('Go Market');
+  });
+
+  test('verify checked tasks', () => {
+    document.body.innerHTML = `
+<main class="main-todo">
+      <h3>Today's To Do <i class="fa-solid fa-arrows-rotate fa-icon"></i></h3>
+      <label for="add" class="add"></label>
+      <input 
+          name="add"
+          id="add"
+          placeholder="Add to your list.."
+      />
+      <ul class="listContainer"></ul>
+      <div class="clearlist">
+          <p class="clear">Clear all completed</p>
+      </div>
+      
+  </main>`;
+    const MyTodo = new MyToDoList();
+    expect(newItem.checkTasks(MyTodo.tasks[0], true)).toBe(true);
+  });
+});
+
+describe('clear completed tasks', () => {
+  test('clear all completed', () => {
+    let myArray = [
+      { index: 1, description: 'Watch a movie', completed: false },
+      { index: 2, description: 'Wash my clothes', completed: false },
+      { index: 3, description: 'Visit a friend', completed: true },
+    ];
+    myArray = myArray.filter((task) => task.completed === false);
+
+    expect(myArray).toHaveLength(2);
+  });
```

Comment:
- [ ] Good job so far, however, please kindly make use of the function which you created for clearing all completed tasks to test the clear all completed functionality. Tests are written to test our functions to ensure they work as expected. So, please test the `Clear all completed` function which you called `clearSelectedTasks` in your index.js.